pmd-core/src/test/java/net/sourceforge/pmd/properties/IntegerPropertyTest.java

```diff
@@ -15,6 +15,7 @@
  *
  * @author Brian Remedios
  */
+@Deprecated
```

Comment:
We should make sure that we cover the testcases in a new unit test class.

Reply:
I think, this is probably covered by NumericConstraintsTest and PropertyDescriptorTest.